src/plugin.ts

```diff
@@ -4,34 +4,54 @@
  * - improve/add some logging
  * - tests
  */
+import fs from 'fs';
 
-const util = require('util');
+import { mkdir, stat, unlink, copyFile} from 'fs/promises';
 
-const fs = require('fs');
-const mkdir = util.promisify(fs.mkdir);
-const stat = util.promisify(fs.stat);
-const unlink = util.promisify(fs.unlink);
-const copyFile = util.promisify(fs.copyFile);
+// stream/promises is only available on node 15+
+// When migrating uncomment the following line
+// import { pipeline } from 'stream/promises';
 
-const { pipeline } = require('stream');
-const pump = util.promisify(pipeline);
+// Delete the following lines when migrating on node 15+
```

Comment:
Are these native for node 15+?

Reply:
Yes, the stream/promises API was added in 15.0.0 
https://nodejs.org/api/stream.html#stream_streams_promises_api